geopandas/explore.py

```diff
@@ -282,7 +282,7 @@ def my_colormap(value):  # scalar value defined in 'column'
     if gdf.crs is None:
         kwargs["crs"] = "Simple"
         tiles = None
-    elif not gdf.crs.equals(4326):
+    elif not gdf.crs == "EPSG:4326":
```

Comment:
The `equals` method was only added in pyproj 2.5 (we currently require 2.2, although we should probably bump that)